docs/getting-started/gs-environments.md

```diff
@@ -13,10 +13,10 @@ The following table lists all current testing and production environments for EU
 
 All EUID endpoints use the same base URL.
 
-| Environment | Base URL |
-| :--- | :--- |
-| Testing | `https://integ.euid.eu/v2` |
-| Production | `https://prod.euid.eu/v2` |
+| Environment | Cloud Region | Code | Base URL |
+| :--- | :--- | :--- | :--- |
+| Integration Testing | AWS EU West (London) | `eu-west-2.eu` | `https://integ.euid.eu/v2` |
```

Comment:
Hi Gen, the code is just `eu-west-2` without the `.eu`

Reply:
@asloobq TYSM. Fixed.